File: I-DataStructures/1-Array/array.c
#include <stdio.h>
#include <string.h>
    
int main(){
  printf("Arrays in C!\n");
  char fruits[5][10] = {"Apples", "Bananas", "Oranges", "Peaches", "Berries"};
  printf("Before: %s\n", fruits[0]);
  strcpy(fruits[0], "Pineapples");
  printf("Before: %s\n", fruits[0]);
  return 0;
}

// gcc array.c
// ./a.out

File: I-DataStructures/1-Array/array.rs
fn main(){
    println!("Arrays in Rust\n");
    let mut fruits: [&str; 5] = ["Apples", "Bananas", "Oranges", "Peaches", "Berries"];
    println!("Before: {}", fruits[0]);
    fruits[0] = "Pineapples";
    println!("After: {}", fruits[0]);
}